desktop/src/de/jjj/dnasic/DesktopLauncher.java

```diff
@@ -2,6 +2,9 @@
 
 import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
 import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
+// prepared import
+//import com.badlogic.gdx.graphics.Texture;
+//import com.badlogic.
```

Comment:
Is this something we are going to need later?
If not I would remove it...

Reply:
i need it in the next step